src/components/PreviewComment/index.js

```diff
@@ -65,7 +65,7 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
             <View style={styles.containerUsername}>
               <CommentUserName isPreviewComment comment={item} user={user} />
               <Gap width={4} />
-              <Dot size={4} color={COLORS.blackgrey} />
+              <Dot size={4} color={COLORS.gray400} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `Dot` component has been changed from `COLORS.blackgrey` to `COLORS.gray400`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could affect readability and contrast.
```diff
- <Dot size={4} color={COLORS.blackgrey} />
+ <Dot size={4} color={COLORS.gray400} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->